leftwm-core/src/handlers/window_handler.rs

```diff
@@ -135,7 +132,7 @@ impl<C: Config, SERVER: DisplayServer> Manager<C, SERVER> {
                 _ => None,
             };
 
-            tracing::debug!("WINDOW CHANGED {:?} {:?}", &window, change);
+            // tracing::debug!("WINDOW CHANGED {:?} {:?}", &window, change);
```

Comment:
My opinion: This can be put aside for the time being since it's called *very* often I'd say, that we should rather create this for the necessary parts/functions otherwise the log file is gonna fill up very fast.